ocw/lib/azure.py

```diff
@@ -97,13 +97,12 @@ def cleanup_all(self):
                 self.cleanup_sle_images_container(block_blob_service, c)
 
     def cleanup_bootdiagnostics_container(self, bbsrv, container):
-        timeout = datetime.now(timezone.utc) + timedelta(hours=24)
         last_modified = container.properties.last_modified
         generator = bbsrv.list_blobs(container.name)
         for blob in generator:
             if (last_modified < blob.properties.last_modified):
                 last_modified = blob.properties.last_modified
-        if (timeout > last_modified):
```

Comment:
in my understanding this does not make sense . **now** will be always bigger than last_modified because **was** written before you have read it :) so it suppose to be bigger on few millisecond. Or I missing somehting ?

Reply:
ah funny, so we actually delete the container only if it is older then 24h.
so this is the minage, or?